quobyte-csi-driver/templates/csi-secrets.yaml

```diff
@@ -0,0 +1,12 @@
+{{ range $storageClass := .Values.quobyte.storageClasses }}
```

Comment:
This is user configuration/cluster specific and does not belong in CSI driver definition. Please use your custom chart(s) to have these templates.

Reply:
Yes I agree; however, I believe having the ability to create StorageClasses together with the quobyte-csi-driver helm-chart is very helpful. I think we could make it an optional during the installation.

Reply:
Unfortunately, that adds one more configuration to the driver. We want to keep the Quobyte CSI driver configuration simple, flexible and non-opinionated (so that many Quobyte users can customize as it suits them) about the user k8s resources. It may fit nicely into your environment but may not fit into all other customers.

If you have use case, please have your own custom chart to deploy your cluster specific resources. We do not want to bloat the CSI driver configuration with user specific k8s resource definitions.